javascript/ql/src/Declarations/SuspiciousMethodName.ql

```diff
@@ -0,0 +1,80 @@
+/**
+ * @name Suspicious method name
+ * @description A method having the name "function", "new", or "constructor"
+ *              is usually caused by a programmer being confused about the TypeScript syntax. 
+ * @kind problem
+ * @problem.severity warning
+ * @id js/suspicious-method-name
+ * @precision high
```

Comment:
I think `very-high` is fine for this query: it seems like we have a pretty good grip on the false positives already, and it is not like we can have spurious flow that mess with our results.

Reply:
Sure? 
2 of the 14 results in DefinitelyTyped were false positives.

Reply:
OK. Just leave it at `high` then.
(you could also say that the query only flags two false positives in all of DefinetelyTyped)